worker/peergrouper/worker.go

```diff
@@ -598,15 +598,10 @@ func (w *pgWorker) updateReplicaSet() (map[string]*replicaset.Member, error) {
 	// Reset machine status for members of the changed peer-group.
 	// Any previous peer-group determination errors result in status
 	// warning messages.
-	// Remove members from the return that are not voters.
 	for id := range desired.members {
 		if err := w.machineTrackers[id].stm.SetStatus(getStatusInfo("")); err != nil {
 			return nil, errors.Trace(err)
 		}
-
-		if !desired.machineVoting[id] {
```

Comment:
Does this have any untended consequences for consumers which aren't the raft clusterer?
It seems slightly concerning we can remove a core piece like this and not require downstream changes elsewhere (besides the raft clusterer which we knew needed fixing).

Reply:
At the moment the only consumers of this information are the raft-transport and raft-clusterer workers.